### PR TITLE
Mark {{anch}} and {{manch}} as deprecated + alternative

### PR DESCRIPTION
**Pull request by teoli2003**
`{{anch}}` and `{{manch}}` are gone from en-US/ and raise a flaw if used. This updates the meta-documentation to mark them as deprecated and listing the alternative.

### CODE REVIEW

File: files/en-us/mdn/structures/macros/other/index.md [hunk 1]
```diff
@@ -76,7 +75,8 @@ These macros have been replace by other ways of doing the same thing, and should
 ### Linking
 
 - The [`SectionOnPage`](https://github.com/mdn/yari/tree/master/kumascript/macros/SectionOnPage.ejs) macro creates a phrase that links to both the name of a section and the article containing that section. For example, `\{{SectionOnPage("/en-US/docs/Mozilla/Firefox/Releases/21", "Changes for Web developers")}}` outputs the following: _{{SectionOnPage("/en-US/docs/Mozilla/Firefox/Releases/21", "Changes for Web developers")}}_.
-- The [`manch`](https://github.com/mdn/yari/tree/master/kumascript/macros/manch.ejs) inserts a link to a method within the current interface; this is intended only for use in interface documentation pages. `\{\{manch("foo")\}\}` produces `<code><a href="current/path#foo">foo()</a></code>` ({{ manch("foo") }}).
+- **Do not use: use a regular Markdown link, eventually with the `coding` style** The [`anch`](https://github.com/mdn/yari/tree/master/kumascript/macros/anch.ejs) macro inserts a link to an anchor. `\{\{Anch("top")\}\}` produces `<a href="#top">top</a>` ({{ Anch("top") }}). You can also add a second parameter which contains replacement text to display as the link text.
```
Comment (dipikabh) [inline files/en-us/mdn/structures/macros/other/index.md:78]:
```suggestion
- **Do not use the [`anch`](https://github.com/mdn/yari/tree/master/kumascript/macros/anch.ejs) macro. It has been deprecated. Instead use a regular Markdown link with the `coding` style.** The [`anch`](https://github.com/mdn/yari/tree/master/kumascript/macros/anch.ejs) macro was used in the past to insert a link to an anchor. `\{\{Anch("top")\}\}` produced `<a href="#top">top</a>` ({{ Anch("top") }}). You could add a second parameter, which contained replacement text to display as the link text.
```

File: files/en-us/mdn/structures/macros/other/index.md [hunk 2]
```diff
@@ -76,7 +75,8 @@ These macros have been replace by other ways of doing the same thing, and should
...
+- **Do not use: use a regular Markdown link, with the `coding` style** The [`manch`](https://github.com/mdn/yari/tree/master/kumascript/macros/manch.ejs) inserts a link to a method within the current interface; this is intended only for use in interface documentation pages. `\{\{manch("foo")\}\}` produces `<code><a href="current/path#foo">foo()</a></code>` ({{ manch("foo") }}).
```
Comment (dipikabh) [inline files/en-us/mdn/structures/macros/other/index.md:79]:
```suggestion
- **Do not use the [`manch`](https://github.com/mdn/yari/tree/master/kumascript/macros/manch.ejs) macro. It has been deprecated. Instead use a regular Markdown link with the `coding` style.** The [`manch`](https://github.com/mdn/yari/tree/master/kumascript/macros/manch.ejs) was used in the past to insert a link to a method within the current interface; this was intended only for use in interface documentation pages. `\{\{manch("foo")\}\}` produced `<code><a href="current/path#foo">foo()</a></code>` ({{ manch("foo") }}).
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hi @teoli2003, thanks for making these updates! I've left some edit suggestions for your consideration.

**Comment by dipikabh** [comment]
@teoli2003, should the 'anch' and 'manch' links that point to the macro files in mdn/yari also be removed because the macro files will be eventually deleted and these links will become invalid?

**Comment by teoli2003** [comment]
> @teoli2003, should the 'anch' and 'manch' links that point to the macro files in mdn/yari also be removed because the macro files will be eventually deleted and these links will become invalid?

I don't think so:
- that way we find the macro quickly
- when they are removed from kumascript, we will remove the whole paragraph here

**Review by dipikabh — APPROVED** [review]

**Pull request merged**